paseos/actors/spacecraft_actor.py

```diff
@@ -19,8 +20,10 @@ class SpacecraftActor(BaseActor):
     # Actor's mass in kg
     _mass = None
 
+    _spacecraft_body_model = None
     _thermal_model = None
     _radiation_model = None
+    _attitude_model = None
```

Comment:
Good point. There are two solutions: 

- We add the possibility to add a control torque (there is a TODO in the code, I can open an issue)
- This can be avoided by disabling attitude disturbances. In this case, there is no tumbling.

Reply:
Why not just a function to setting the current pointing? Super simple to implement and allows you implement more complex interaction via custom parameters etc.

Reply:
(please don't resolve if I should read because it will be hidden :P)

I would suggest to move the attitude into actor (maybe even base_actor?) since pointing / attitude of the actor can be interesting even if it is not disturbed or maybe even for ground station (think ground station antenna pointing)?

Reply:
Sorry, I am resolving stuff because there are so many comments that I lose track of what I have to implement xD 
Yes, I think it's smart to move it to base.